src/components/AnnotationStylePopup/AnnotationStylePopup.js

```diff
@@ -51,6 +51,7 @@ class AnnotationStylePopup extends React.Component {
     return(
       <div className={className} data-element="annotationStylePopup" onClick={() => closeElement('annotationPopup')}>
         <StylePopup 
+          annotationType = {annotation.Subject}
```

Comment:
Using `annotation.Subject` to get the annotation type is ok, but we need to be careful here :^). 
In the new UI, when an annotation is created, the value of its `Subject` depends on what language the user is using. Currently the UI supports 4 languages, which can be found under `i18n` folder, and a user can change the language by calling readerControl.setLanguage(language). 
That being said, if you called readerControl.setLanguage('fr'), and then select the highlight tool button to create an highlight annotation, the `Subject` of that annotation will be Surligne instead of Highlight. If possible, we should try to use other property than `Subject`